doc/user_guide/conf.py

```diff
@@ -372,11 +372,11 @@
 linkcheck_anchors_ignore = ['user-content-netcdf-library-lfric-examples']
 
 # MyBinder fails on a very regular basis so we skip those links.
-# The puma site no longer exists but the GOcean documentation needs to be
-# updated (TODO #237).
-# Despite its name 'collab.metoffice.../twiki' requires authenticated access.
 linkcheck_ignore = [r'^https://mybinder.org/v2/gh/stfc/psyclone',
-                    r'https://puma.nerc.ac.uk/trac/GOcean',
+# Shpinx has problems wiht Github anchors, so we skip the links to anchors to
+# the main README
+                    r'^https://github.com/stfc/PSyclone#',
+# Despite its name 'collab.metoffice.../twiki' requires authenticated access.
```

Comment:
To be fair to them, it's no longer called 'collab' so we could just shorten (to "Requires authentication") but I'll probably do that if I'm happy with the rest of the PR.

Reply:
Done